code/modules/integrated_electronics/subtypes/memory.dm

```diff
@@ -40,8 +40,15 @@
 		O.push_data()
 	activate_pin(2)
 
+/obj/item/integrated_circuit/memory/tiny
+	name = "Small memory circuit"
+	desc = "This circuit can store two pieces of data."
+	icon_state = "memory4"
+	power_draw_per_use = 2
+	number_of_pins = 2
+
 /obj/item/integrated_circuit/memory/medium
-	name = "memory circuit"
+	name = "Medium memory circuit"
```

Comment:
Because of BYOND, names should be lowercase if they're not a proper noun, or add a ``\improper`` at the start.  Would rather have it be lowercase to be consistant with the rest of the circuits and items in-game.